Add empty_room.yaml for spatial navigation training

sandguine opened the pull request:
This adds a new training config with large 60x60+ rooms, no distractor objects, and 5–10 randomly scattered altars. The purpose is to support spatial navigation learning in minimally structured environments. Based on the emptyspace_sparse setup but with larger room sizes, increased mean object distance, and no other objects besides altars.

daphnedemekas commented:
Hey Sandy! This looks good. We decided today to change to having 4 agents total so that we can have single-agent maps for our bigger map, so if you could change 20 to 4, and have 4 rooms (one agent per room, like you have now, that would be great). 

Then, can you please:

- pull from george-navigation-training into your branch 
- make a new file called env/mettagrid/navigation/training/multienv_emptyroom.yaml and set up a multienv environment that has varied_terrain, terrain_from_numpy, and your empty room yaml file in it. 
- You can copy the current multienv.yaml to see how to do that 

Once you've done that and pushed those changes to your branch, we can get you set up running training on this environment! 

The command for that will be 

`python -m devops.aws.batch.launch_task --cmd=train --run=b.$USER.RUN_NAME  --git-branch=YOUR_BRANCH_NAME --mettagrid-branch=george-navigation-training wandb.enabled=true wandb.track=true trainer.env=env/mettagrid/navigation/training/ENV_NAME --gpus=4 --node-gpus=4 
`

Come to me for any questions!

sandguine commented:
Updated this PR with the following changes:

- Updated all configs to use 4 agents, aligned with single-agent-per-room structure
- Pulled latest changes from `george-navigation-training`
- Created `multienv_emptyroom.yaml` combining varied_terrain, terrain_from_numpy, and empty_room

Ready for training runs once merged.
